### PR TITLE
add new translation setup in french

### PR DESCRIPTION
**Pull request by TheLynk**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

ex. "MyGame4: implement new game" or "Docs: add new guide for customizing MyGame3"

## What is this fixing or adding?
Adding new translation for minecraft setup in french and OOT setup in french

## How was this tested?
in local web server

## If this makes graphical changes, please attach screenshots.
no graphical changes

### CODE REVIEW

**Comment by Berserker66** [comment]
@KonoTyran and @espeon65536 
My recommendation for review is to run it back to english through a translation and see if it looks at least mostly ok. Unless one of you is capable of French of course.

**Comment by espeon65536** [comment]
I do not understand French, but I am suspicious of the translation quality because the example YAML provided in the OoT tutorial (which needs to be gotten rid of anyway, tbh) is also translated into French. This will obviously not work for generation, which a human translating should recognize; this makes me suspect the entire thing was done by machine translation.

**Comment by TheLynk** [comment]
I did use translation software but I reformulated the sentences that had no no or badly translated while keeping the information

**Comment by TheLynk** [comment]
So I just finished making the necessary changes for the example in the YAML for OOT

File: worlds/oot/docs/setup_fr.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+  Ocarina of Time: 1
+requires:
+  version: 0.1.7 # Version d'Archipelago requise pour que ce yaml fonctionne comme prévu.
+# Options partagées prises en charge par tous les jeux :
+accessibility:
+  items: 0 # Garantit que vous pourrez acquérir tous les articles, mais vous ne pourrez peut-être pas accéder à tous les emplacements
+  locations: 50 # Garantit que vous pourrez accéder à tous les emplacements, et donc à tous les articles
+  none: 0 # Garantit seulement que le jeu est battable. Vous ne pourrez peut-être pas accéder à tous les emplacements ou acquérir tous les objets
+progression_balancing: # Un système pour réduire le BK, comme dans les périodes où vous ne pouvez rien faire, en déplaçant vos éléments dans une sphère d'accès antérieure
+  0: 0 # Choisissez un nombre inférieur si cela ne vous dérange pas d'avoir un multimonde plus long, ou si vous pouvez glitch / séquences autour d'éléments manquants.
```
Comment (Marechal-L) [inline worlds/oot/docs/setup_fr.md:74]:
```suggestion
  0: 0 # Choisissez un nombre inférieur si cela ne vous dérange pas d'avoir un multimonde plus long, ou si vous pouvez glitch / faire du hors logique.
```

**Comment by Marechal-L** [comment]
I suggest to make this one last change to make it mergeable.
Thanks from the French players community :)

**Comment by TheLynk** [comment]
> I suggest to make this one last change to make it mergeable. Thanks from the French players community :)

to welcome and update my git for to make it mergeable

**Review by black-sliver — CHANGES_REQUESTED** [review]
I don't speak french, but I believe all requested changes were made and two people read it.

If something is or becomes horribly wrong or outdated content-wise, we'll hopefully get a bug report for it.

The only thing that is missing now is adding it to the worlds' tutorials
[here](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/oot/__init__.py#L88) and [here](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/minecraft/__init__.py#L50) (scroll up a bit to see what espanol does and copy that).

**Comment by TheLynk** [comment]
I made the changes that you required

File: worlds/minecraft/docs/minecraft_fr.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Guide de configuration du randomiseur Minecraft
+
+## Logiciel requis
+
+-Minecraft Java Edition à partir de
```
Comment (black-sliver) [inline worlds/minecraft/docs/minecraft_fr.md:5]:
```suggestion
- Minecraft Java Edition à partir de
```

File: worlds/oot/docs/setup_fr.md [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+
+### Connectez-vous au multiserveur
+
+Une fois le client et l'émulateur démarrés, vous devez les connecter. Dans l'émulateur, cliquez sur "Outils"
+menu et sélectionnez "Console Lua". Cliquez sur le bouton du dossier ou appuyez sur Ctrl+O pour ouvrir un script Lua.
+
+Accédez à votre dossier d'installation Archipelago et ouvrez `data/lua/OOT/oot_connector.lua`.
+
+Pour connecter le client au multiserveur, mettez simplement `<adresse>:<port>` dans le champ de texte en haut et appuyez sur Entrée (si le
+le serveur utilise un mot de passe, saisissez dans le champ de texte inférieur `/connect <adresse> :<port> [mot de passe]`)
```
Comment (black-sliver) [inline worlds/oot/docs/setup_fr.md:420]:
```suggestion
le serveur utilise un mot de passe, saisissez dans le champ de texte inférieur `/connect <adresse>:<port> [mot de passe]`)
```

**Review by black-sliver — CHANGES_REQUESTED** [review]
Found two formatting issues

**Review by black-sliver — APPROVED** [review]
Thanks!

**Pull request merged**